opensquirrel/exporter/quantify_scheduler_exporter.py

```diff
@@ -39,13 +38,15 @@ def visit_bloch_sphere_rotation(self, g: BlochSphereRotation) -> None:
             # Rxy rotation.
             theta = round(math.degrees(g.angle), FIXED_POINT_DEG_PRECISION)
             phi: float = round(math.degrees(math.atan2(g.axis[1], g.axis[0])), FIXED_POINT_DEG_PRECISION)
-            self.schedule.add(quantify_scheduler_gates.Rxy(theta=theta, phi=phi, qubit=self._get_qubit_string(g.qubit)))
+            self.schedule.add(
+                quantify_scheduler_gates.Rxy(theta=theta, phi=phi, qubit=self._get_qubit_string(Qubit(g.qubit)))
```

Comment:
Can we change `_get_qubit_string` to use a `QubitLike` parameter? So that we don't need the conversion to `Qubit` here either.